sql-statements/sql-statement-alter-table.md

```diff
@@ -65,38 +61,81 @@ EXPLAIN SELECT * FROM t1 WHERE c1 = 3;
 3 rows in set (0.00 sec)
 ```
 
+用户可使用 [`ALTER TABLE .. ADD INDEX`](/sql-statements/sql-statement-add-index.md) 语句在 `t1` 表上添加索引。添加后，`EXPLAIN` 的分析结果显示 `SELECT * FROM t1 WHERE c1 = 3;` 查询已使用效率更高的索引范围扫描：
+
 {{< copyable "sql" >}}
 
 ```sql
 ALTER TABLE t1 ADD INDEX (c1);
+EXPLAIN SELECT * FROM t1 WHERE c1 = 3;
+```
+
+```sql
++------------------------+---------+-----------+------------------------+---------------------------------------------+
```

Comment:
Is the `Query OK, 0 rows affected (0.30 sec)` line omitted on purpose? I see it in https://docs.pingcap.com/tidb/dev/sql-statement-alter-table

Reply:
Oh, I missed it. Added in https://github.com/pingcap/docs-cn/pull/4680/commits/df851d5bf89d68238d42650ac5983fa4f1243e75.